cbScrollTo/js/cargobay.scroll-to.js

```diff
@@ -0,0 +1,98 @@
+/* ==========================================================================
+   Scroll to
+
+   Initialize:
+   cargobay.scrollTo();
+
+   Support:
+   Latest Chrome
+   Latest FireFox
+   Latest Safari
+   IE10 and up
+   ========================================================================== */
+
+var cargobay = cargobay || {};
+
+cargobay.scrollTo = function(undefined) {
+
+    // Set up global variables and functions
+    var init, duration, _scrollTo;
+
+    // Smooth scroll using easeInOutQuad curve
+    Math.easeInOutQuad = function (t, b, c, d) {
```

Comment:
I'm not a big fan of adding your own methods to native javascript objects. Does anybody else have any problems with this? 

The indentation is not correct, it should be 4 spaces instead of 2.
Ps: If code gets copied from somewhere else, maybe we should put a comment giving the original author some credit (https://gist.github.com/james2doyle/5694700).